docs/release-notes/8.6.asciidoc

```diff
@@ -1,6 +1,16 @@
 [[release-notes-header-8.6.0]]
 == 8.6
 
+[discrete]
+[[release-notes-8.6.1]]
+== 8.6.1
+
+[discrete]
+[[bug-fixes-8.6.1]]
+==== Bug fixes and enhancements
+* Fixes a bug that impacted the way Osquery results displayed in Lens and Discover ({pull}148260[#148260]).
```

Comment:
is this ''the way osquery results displayed in'' a proper form? Or is it a typo? (sorry, still learning English ;p) 

I thought of 'osquery results were displayed' / 'osquery displayed results' .

Reply:
So "Osquery results were displayed" works because it's almost like you're saying "Fixes a bug that impacted the way Osquery results would be displayed in Lens and Discover". I think both versions of the sentence work, but I have no objections to adding "were" in you think it improves the sentence flow and clarity. :)

Reply:
```suggestion
* Fixes a bug that impacted the way Osquery results were displayed in Lens and Discover ({pull}148260[#148260]).
```